packages/nimble-components/package.json

```diff
@@ -98,6 +101,7 @@
     "@storybook/html-webpack5": "^7.0.4",
     "@storybook/theming": "^7.0.4",
     "@types/jasmine": "^4.3.1",
+    "@types/markdown-it": "^12.2.3",
```

Comment:
The types for our dependencies are versioned with that dep. Can you move this to the dependencies section?

Reply:
Moved `@types/markdown-it` to the dependencies section.